docker/benchmarks/run-server.sh

```diff
@@ -5,6 +5,13 @@ set -x
 
 server_ip=$(ip route get 1 | awk '{print $NF;exit}')
 
+if [ -e /var/log/waagent.log ]
```

Comment:
Why not look for this file in the `BenchmarkServer.Startup` constructor? Seems inconsistent and extra complicated to require the value to be passed in.

Reply:
`BenchmarkServer` is running inside a container, so it can't detect whether the host is `Cloud` or `Physical`.  I couldn't find a better way than requiring a command-line parameter.

Reply:
Ick. Guess the option to conditionally mount this directory in the `docker run` command is just as ugly -- more complex here balanced against only slight `BenchmarkServer.Startup` cleanup.